webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala

```diff
@@ -3394,4 +3433,274 @@ class OntologyV2R2RSpec extends R2RSpec {
       freetestLastModDate = responseAsInput.ontologyMetadata.lastModificationDate.get
     }
   }
+
+  "create a class that is a sequence of a video resource" in {
```

Comment:
The tests you added are only testing positive cases. Would it be worth to add tests which verify that something fails as expected?

Reply:
there isn't really any expected failure case: `isSequenceOf` can be used on any resource, then it just "doesn't have a special meaning". So I could only test cases that fail on any property (like "not add it to a resource that doesn't exist"), and I don't think that would make much sense to do.